Source/ALS/Public/Settings/AlsAnimationInstanceSettings.h

```diff
@@ -41,6 +41,9 @@ class ALS_API UAlsAnimationInstanceSettings : public UDataAsset
 	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Settings")
 	FAlsGeneralTurnInPlaceSettings TurnInPlace;
 
+	UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Settings|Ragdolling")
+	float RagdollingStartBlendTime{0.25f};
+
```

Comment:
I defined the value used in the ‘Blend Poses by Gameplay Tag’ of AB_Als_Ragdolling’s Ragdolling Blend Time as a setting value because I wanted to refer to it in AlsCharacter as well.

The GetUp AnimMontage will abruptly warp the pose unless the blend to Ragdolling by ‘Blend Poses by Gameplay Tag’ is finished, as the start blend time is set to 0.